### DISCUSSION TITLE
在软路由上运行的最新iptv-checker-rs

### DISCUSSION THREAD

**Discussion by mftvbox** [zhimin-dev/iptv-checker-rs#12] [General]
/mnt/sata2-1/iptv-checker/iptv-checker check -i https://iptv.002397.xyz/fmml_ipv6.m3u -c 10 -t 5000 -o /www/mytv.txt
您输入的文件地址是: https://iptv.002397.xyz/fmml_ipv6.m3u
您输入是：标准格式m3u格式文件
出文件: .//www/mytv.txt
文件中源总数： 92
检查进度: 92/92thread 'main' panicked at src/common/m3u.rs:334:63:
called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

这都检查完了，咋还出这个错误，完全看不懂了！！（在软路由上运行的）

**Comment by mftvbox** [comment]
![出错误](https://github.com/zhimin-dev/iptv-checker-rs/assets/142022126/30289157-4b75-48fa-8e58-ae9000d56ed2)

**Comment by owocao** [comment]
同样的问题，不知道啥意思。。。

**Comment by zmisgod** [comment]
看起来像是输出的文件打不开，晚点我istore中试试
